src/libnpf/npf.c

```diff
@@ -259,7 +259,7 @@ _npf_config_consdict(prop_dictionary_t npf_dict)
 nl_config_t *
 npf_config_retrieve(int fd)
 {
-	prop_dictionary_t npf_dict;
+	prop_dictionary_t npf_dict = NULL;
```

Comment:
I don't mind adding these, but it's a false positive (like the other ones below).